### ISSUE TITLE
A lot of problems after Official AdonisJS v5 release.

### ISSUE THREAD

**Issue by dawidow** [wahyubucil/adonis-mix-asset#5]
Hey bro, I love your laravel-mix wrapper for AdonisJS. Can you update your package? At this moment this throw us a lot of errors while installing it in a new project. Please <3

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: adonisapp@1.0.0
npm ERR! Found: @adonisjs/view@6.0.1
npm ERR! node_modules/@adonisjs/view
npm ERR!   @adonisjs/view@"^6.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @adonisjs/view@"^3.1.0" from adonis-mix-asset@2.3.0
npm ERR! node_modules/adonis-mix-asset
npm ERR!   dev adonis-mix-asset@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/dawidowczarz/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dawidow/.npm/_logs/2021-05-14T15_47_09_190Z-debug.log```

**Comment by wahyubucil** [comment]
Hi Dawid, I'll release a new version that compatible with the official AdonisJS v5 release. I'm working on it this week. Thanks!

**Comment by dawidow** [comment]
@wahyubucil I love what you do, and I look forward to it. This is the number one (must have) package for me in AdonisJS. Thanks a lot, regards

**Comment by wahyubucil** [comment]
@dawidow I've released v3.0.0 that compatible with the latest AdonisJS v5 release. If it does solve your issue, feel free to close this issue. Regards

**Comment by dawidow** [comment]
@wahyubucil thanks bro, it's works! Have a good day
